### ISSUE TITLE
Template: Attributes should support binding to non-Strings and just use toString()

### ISSUE THREAD

**Issue by matanlurey** [angular/angular#2968]
Otherwise, for something like `[attr.aria-invalid]="isInvalid"` you must do something like:

```dart
bool _isInvalid;
String get isInvalid => _isInvalid.toString();
```

**Comment by mhevery** [comment]
Why not just write

```
<div attr.aria-invalid="{{ isInvalid }}"></div>
```

http://plnkr.co/edit/G1sPP6pspqH4GJ50sziC?p=preview

**Comment by mhevery** [comment]
It seem like both types of bindings work as expected. 

`<div attr.aria-title="{{ [1+2] }}" [attr.aria-foo]="[1+2]">inspect me</div>`

Is this a Dart only issue?

Closing as I can't replicate. Reopen if `<div attr.aria-invalid="{{ isInvalid }}"></div>` does not work on Dart.

**Comment by matanlurey** [comment]
I had no idea interpolation even worked inside of attributes in Angular 2.
